### PR TITLE
fix(launcher): updates gitHub Auth reload flow to authorize uses if a…

### PR DESCRIPTION
**Pull request by invincibleJai**
### Launcher
- updates gitHub Auth reload flow to authorize uses if already authorized GitHub with OSIO

### CODE REVIEW

**Comment by invincibleJai** [comment]
@dlabrecq Hi Dan, pls review.

**Comment by invincibleJai** [comment]
@joshuawilson pls review, thanks :)

File: src/app/space/app-launcher/services/app-launcher-gitprovider.service.ts [hunk 1]
```diff
@@ -119,12 +119,12 @@ export class AppLauncherGitproviderService implements GitProviderService {
             this.gitHubUserLogin = user.login;
             orgs.push(this.gitHubUserLogin);
             let gitHubDetails = {
-              authenticated: this.isPageRedirect() ? true : false,
+              authenticated: true,
               avatar: user.avatarUrl,
               login: user.login,
               organizations: orgs
             } as GitHubDetails;
-            return this.isPageRedirect() ? Observable.of(gitHubDetails) : Observable.empty();
```
Comment (dlabrecq) [inline src/app/space/app-launcher/services/app-launcher-gitprovider.service.ts]: We should reomove the private isPageRedirect method now

Comment (invincibleJai) [reply src/app/space/app-launcher/services/app-launcher-gitprovider.service.ts]: @dlabrecq thanks Dan, have updated it here https://github.com/fabric8-ui/fabric8-ui/pull/2701

**Pull request merged**